# This is a message for emulated Pixhawk HW
std_msgs/Header header
float64 thrusters_power
float64 batt_capacity_remaining
float64 batt_voltage_remaining
float64 batt_charge_remaining
float32[] rpm 